doc/lua_api.txt

```diff
@@ -9700,6 +9700,8 @@ section, along with the datatypes they accept.
 All of the properties in this list can be animated with `*_tween` tables
 unless otherwise specified. For example, `jitter` can be tweened by setting
 a `jitter_tween` table instead of (or in addition to) a `jitter` table/value.
+In this section, a float range is a table defined as so: { min = x, max = y }
+X and Y are your supplimented values.
```

Comment:
It would be better to have a list of types specific to particlespawner animations plus an example. Also `vec3` is not an established Minetest term and hence needs better explanation (is it a normal `vector`?).

Reply:
Pretty sure what a range is is defined at some point. If not you can infer it from the vec3 ranges.

Reply:
I inferred it after a few mistakes, but they didn't.